packages/nextjs/app/builders/[0x95E08FA8ac4301acC5b943f860Cd8AC84433e3CF]/page.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import type { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <div className="flex flex-col items-center md:flex-row justify-center md:h-[80vh] md:gap-4 ">
+      <div className="flex flex-col items-center mb-4 md:mb-0">
+        <img
```

Comment:
You could add : 

`{/* eslint-disable-next-line @next/next/no-img-element */}`

To fix gh-actions failing, we could have use Next Image component but then we need to add remote pattern to `next.config` maybe lets ignore it for now 🙌

Reply:
ya i added it initially, to the next.config. but reverted it. If a lot of people in the batch are using it from the same source, I can add it back.